build/build-mrt.yml

```diff
@@ -55,6 +55,13 @@ steps:
     command: 'custom'
     arguments: 'restore ${{ parameters.MRTSourcesDirectory }}\mrt\mrm\unittests\packages.config -ConfigFile nuget.config -PackagesDirectory ${{ parameters.MRTSourcesDirectory }}\mrt\packages'
 
+# NuGetCommand@2
+- task: 333b11bd-d341-40d9-afcf-b32d5ce6f23b@2
+  displayName: 'NuGet restore of WindowsAppSDKInsightsResource'
```

Comment:
I am not sure if every project should reference resource lib. Updated the wiki to capture what should be included.

Reply:
Agreed. I'm not adding it as a reference to every project. This is just running nuget restore.

Reply:
Correct. This is not the latest. I have updated the script so that the restore takes place in the main script (ProjectReunion-BuildFoundation.yml). This script is replaced with the restore of TraceLoggingConfig.h. Please rebase.